packages/lib/src/types.dom.ts

```diff
@@ -855,6 +855,13 @@ interface SvgElementAttributes {
     y?: string | number
   }
   text: SvgStrokeAttributes & {
+    x: string
+    y: string
+    dx: string
+    dy: string
+    rotate: "none" | (string & {})
```

Comment:
let me know if this is ok

Reply:
that's cool - I think we need to make these new properties optional, though - and `x`, `y` etc could be `string | number` afaik?

Reply:
oh shoot, good catch, I thought I made them optional. Must have done that locally. brb will push in a sec

Reply:
Made optional and made `string | number` as indicated in the following resources
[attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text#attributes) using `x` as reference denotes it is type [length](https://developer.mozilla.org/en-US/docs/Web/SVG/Content_type#length) & 
[percent](https://developer.mozilla.org/en-US/docs/Web/SVG/Content_type#percentage)